apps/prairielearn/src/pages/home/home.html.ts

```diff
@@ -199,17 +200,28 @@ function InstructorCoursesCard({ instructorCourses }: { instructorCourses: Instr
                       </a>`
                     : `${course.short_name}: ${course.title}`}
                 </td>
-                <td>
+                <td class="js-course-instance-list">
                   ${course.course_instances.map(
                     (course_instance) => html`
                       <a
-                        class="btn btn-outline-primary btn-sm my-1"
+                        class="btn btn-outline-primary btn-sm my-1 ${course_instance.expired
+                          ? 'd-none js-inactive-course-instance'
+                          : ''}"
                         href="${config.urlPrefix}/course_instance/${course_instance.id}/instructor"
                       >
                         ${course_instance.long_name}
                       </a>
                     `,
                   )}
+                  ${course.course_instances.some((ci) => ci.expired)
+                    ? html`<button
+                        class="btn btn-light btn-xs my-1"
```

Comment:
What do you think about using `btn-outline-dark` here instead? I think it makes it a little more obvious that it's clickable.

<img width="523" alt="Screenshot 2025-04-14 at 14 05 20" src="https://github.com/user-attachments/assets/4a664f27-07b3-4707-bd72-7c81ba590c4f" />


```suggestion
                        class="btn btn-outline-dark btn-xs my-1"
```

Reply:
Done in 34d113db89b8421d6f320c1f0edb0f61d9063b55.